app/assets/stylesheets/modules/_wizard.styl

```diff
@@ -97,6 +97,10 @@ wizard-width = 860px
           img
             height 20px
             width 20px
+      span
```

Comment:
.wizard is massively overused. Instead of adding this span styling here, where it will affect every span that has the `wizard` class (in lots of different places), you should put this in _campaign.styl, and possibly add a new class to the haml like `campaign-edit`, and make the span styling only apply to that.